sensehat/src/main/java/com/google/android/things/contrib/driver/sensehat/BaroTemp.java

```diff
@@ -0,0 +1,170 @@
+package com.google.android.things.contrib.driver.sensehat;
+
+import com.google.android.things.pio.I2cDevice;
+import com.google.android.things.pio.PeripheralManager;
+
+import java.io.IOException;
+
+/**
+ * This class allows access to the LPS25H on the SenseHat.
+ * <p>
+ * See also: https://www.pololu.com/file/download/LPS25H.pdf?file_id=0J761</p>
+ * <p>Source code referenced: https://github.com/tkurbad/mipSIE/blob/master/python/AltIMU-10v5/i2c.py</p>
+ */
+public class BaroTemp implements AutoCloseable {
```

Comment:
Generally our class names closely reflect the actual name of the part, this being the LPS25H. Then the comments in the class would describe that it is a temperature/barometric pressure sensor.